src/lib/coda_lib/coda_lib.ml

```diff
@@ -662,7 +662,8 @@ module Make (Inputs : Inputs_intf) = struct
   let create (config : Config.t) =
     trace_task "coda" (fun () ->
         let external_transitions_reader, external_transitions_writer =
-          Linear_pipe.create ()
+          Strict_pipe.create Synchronous
+          (* Linear_pipe.create () *)
```

Comment:
remove

Reply:
Just to be sure, by `remove' you mean remove the comments here.